cabot/cabotapp/models.py

```diff
@@ -894,6 +910,7 @@ def prefixed_mobile_number(self):
 
     mobile_number = models.CharField(max_length=20, blank=True, default='')
     hipchat_alias = models.CharField(max_length=50, blank=True, default='')
+    mattermost_alias = models.CharField(max_length=50, blank=True, default='')
```

Comment:
Why do we need a `mattermost_alias` field on `UserProfile`? Shouldn't it just be in `MatterMostAlertUserData`?

(I think the `mobile_number` and `hipchat_alias` fields here are deprecated)